api/schemas/resource_type.json

```diff
@@ -1,43 +0,0 @@
-{
```

Comment:
Oh wait, should I not have removed this? In puppet server it looks like we use this in our catalog schema, but here it looks like the [catalog.json](https://github.com/puppetlabs/puppet/blob/master/api/schemas/catalog.json#L33) just has all the same stuff in it.